src/group_theory/nilpotent.lean

```diff
@@ -249,6 +258,20 @@ def lower_central_series (G : Type*) [group G] : ℕ → subgroup G
 
 variable {G}
 
+@[simp] lemma lower_central_series_zero_def : lower_central_series G 0 = ⊤ := rfl
+
+lemma mem_lower_central_series_succ_iff (n : ℕ) (x : G) : x ∈ lower_central_series G (n + 1) ↔
+  x ∈ closure {x | ∃ (p ∈ lower_central_series G n) (q ∈ (⊤ : subgroup G)), p * q * p⁻¹ * q⁻¹ = x}
+:= iff.rfl
```

Comment:
style: `:=` usually goes on the end of a line, and we usually don't wrap the result without first wrapping after `:`:
```suggestion
lemma mem_lower_central_series_succ_iff (n : ℕ) (x : G) :
  x ∈ lower_central_series G (n + 1) ↔
    x ∈ closure {x | ∃ (p ∈ lower_central_series G n) (q ∈ (⊤ : subgroup G)), p * q * p⁻¹ * q⁻¹ = x} :=
iff.rfl
```